web/src/main/java/org/springframework/security/web/server/util/matcher/ServerWebExchangeMatchers.java

```diff
@@ -68,12 +66,7 @@ public static ServerWebExchangeMatcher matchers(ServerWebExchangeMatcher... matc
 	public static ServerWebExchangeMatcher anyExchange() {
 		// we don't use a lambda to ensure a unique equals and hashcode
```

Comment:
It might be right here that's the issue

Reply:
I've added a suppression, so IntelliJ shouldn't suggest the conversion to lambda here anymore.